package.json

```diff
@@ -27,7 +27,7 @@
     }
   },
   "lint-staged": {
-    "*": [
+    "*.{js,jsx,ts,tsx}": [
```

Comment:
Can you explain what this does and why you decided to add this in this PR?

Reply:
I couldn't add files to my commit because prettier & eslint were running against unsupported file types and complained about errors.

Reply:
Let's solve this issue with the configuration of these respective tools. Maybe the way we run prettier in the CI doesn't read `prettier.config.js`? If you want to go ahead with this quick fix, could you also create a bug issue to fix it in the right place?

Reply:
Great, let's do so. 
https://github.com/xFAANG/askql/issues/221